src/Microsoft.AspNetCore.Routing/Template/TemplateBinder.cs

```diff
@@ -90,8 +126,8 @@ public TemplateValuesResult GetValues(RouteValueDictionary ambientValues, RouteV
                     }
                 }
 
-                if (!hasNewParameterValue && 
-                    !hasCurrentParameterValue && 
+                if (!hasNewParameterValue &&
+                    !hasCurrentParameterValue &&
```

Comment:
I don't think there's any reason to change this class. In the rare case that someone is constructing it themselves, they can just pass in the correct encoder.

Reply:
Why is this class still changing? While users _could_ construct an instance, it's primarily constructed in `TreeRouter` and `RouteBase`.